ApplyToTheIndustry/Assets/Scripts/AI Grader/Grader.cs

```diff
@@ -0,0 +1,105 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Grader : MonoBehaviour
+{
+     Dictionary<string, int> skillPoints = new Dictionary<string, int>();
+     HashSet<string> countedSkills = new HashSet<string>();
+     int Totalpoints =0;
+
+    void Start()
+    {
+        
+        GetListOfSkills();
+
+    }
+
+    void Update()
```

Comment:
For now checking stuff in Update is fine, but the grader will probably work on a passive manner, meaning it will likely be called to evaluate something rather than checking every frame.